web/themes/joinup/prototype/js/script.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
We are probably not going to use this script (entirely or part of it), especially since we intend to always show the "Load more" button in the last part of the row of 3 items. If there are less than 3 items (including the Load more button) that means that there are no more items to be shown and we do not show the button.